{
  "model": "gpt-5-medium",
  "service": "github",
  "task": "harmony__fix_conflict",
  "runs": {
    "run-1": {
      "success": false,
      "error_message": null,
      "execution_time": 986.9666724205017,
      "token_usage": {
        "input_tokens": 1590965,
        "output_tokens": 31097,
        "total_tokens": 1622062,
        "reasoning_tokens": 28160
      },
      "turn_count": 25
    },
    "run-2": {
      "success": true,
      "error_message": null,
      "execution_time": 496.9921624660492,
      "token_usage": {
        "input_tokens": 1611447,
        "output_tokens": 22916,
        "total_tokens": 1634363,
        "reasoning_tokens": 20480
      },
      "turn_count": 25
    },
    "run-3": {
      "success": true,
      "error_message": null,
      "execution_time": 505.54674315452576,
      "token_usage": {
        "input_tokens": 1848394,
        "output_tokens": 22672,
        "total_tokens": 1871066,
        "reasoning_tokens": 20224
      },
      "turn_count": 27
    },
    "run-4": {
      "success": true,
      "error_message": null,
      "execution_time": 1089.6910638809204,
      "token_usage": {
        "input_tokens": 2169929,
        "output_tokens": 45702,
        "total_tokens": 2215631,
        "reasoning_tokens": 42240
      },
      "turn_count": 33
    }
  }
}